### PR TITLE
⭐️ add vulnerability policy for VMware vCloud

### PR DESCRIPTION
**Pull request by chris-rock**
Adds the MQL check for the disabled slpd service, as recommended in https://www.vmware.com/security/advisories/VMSA-2021-0002.html

### CODE REVIEW

File: core/mondoo-openssl-vulnerability.mql.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 policies:
   - uid: mondoo-openssl-vulnerability
-    name: OpenSSL Vulnerability Policy by Mondoo
+    name: Mondoo OpenSSL Vulnerability Policy
```
Comment (tas50) [inline core/mondoo-openssl-vulnerability.mql.yaml:3]: `by Mondoo` is how we've named all our other policies. We should change this one back or change all the other ones

File: core/mondoo-vmware-vulnerability.mql.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+policies:
+  - uid: mondoo-vmware-vulnerability
+    name: Mondoo VMware vCenter Vulnerability Policy
```
Comment (tas50) [inline core/mondoo-vmware-vulnerability.mql.yaml:3]:
```suggestion
    name: VMware vCenter Vulnerability Policy by Mondoo
```

File: core/mondoo-vmware-vulnerability.mql.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Mondoo, Inc
+        email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: vmware,vmware-vsphere
+      mondoo.com/category: security
+    docs:
+      desc: |
+        ## Overview
+
+         Mondoo OpenSSL VMware vCenter Policy checks for vulnerable vCenter/ESXi configuration. It should be used in combination with Mondoo's Platform Vulnerability Management to identify missing patches.
```
Comment (tas50) [inline core/mondoo-vmware-vulnerability.mql.yaml:15]:
```suggestion
        OpenSSL VMware vCenter Policy by Mondoo checks for vulnerable vCenter/ESXi configuration. It should be used in combination with Mondoo's Platform Vulnerability Management to identify missing patches.
```

File: core/mondoo-openssl-vulnerability.mql.yaml (hunk 1 above)
Comment (chris-rock) [reply core/mondoo-openssl-vulnerability.mql.yaml:3]: Thank you for the hint. I copied this from cnquery. I align it there.

File: core/mondoo-vmware-vulnerability.mql.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Mondoo, Inc
+        email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: vmware,vmware-vsphere
+      mondoo.com/category: security
+    docs:
+      desc: |
+        ## Overview
+
+        Mondoo OpenSSL VMware vCenter Policy checks for vulnerable vCenter/ESXi configuration. It should be used in combination with Mondoo's Platform Vulnerability Management to identify missing patches.
```
Comment (tas50) [inline core/mondoo-vmware-vulnerability.mql.yaml:15]: What is `Mondoo's Platform Vulnerability Management`?

Comment (chris-rock) [reply core/mondoo-vmware-vulnerability.mql.yaml:15]: Screenshot 2023-02-08 at 00 52 37

Comment (tas50) [reply core/mondoo-vmware-vulnerability.mql.yaml:15]: Shouldn't that be calling out the policy name then?

Comment (chris-rock) [reply core/mondoo-vmware-vulnerability.mql.yaml]: Agreed. Thank you @tas50

**Review by atomic111 — APPROVED** [review]
@chris-rock thanks for the new vmware vuln policy

**Pull request merged**